cmd/res/devices/camera.yaml.example

```diff
@@ -0,0 +1,27 @@
+# Pre-defined Devices
+DeviceList:
```

Comment:
gopkg.in/yaml.v3 lib is case sensitive, so the YAML file will not unmarshal as expected.
https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/dtos/device.go
```
type Device struct {
	DBTimestamp    `json:",inline"`
	Id             string                        `json:"id,omitempty" yaml:"id,omitempty" validate:"omitempty,uuid"`
	Name           string                        `json:"name" yaml:"name" validate:"required,edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
	Description    string                        `json:"description,omitempty" yaml:"description,omitempty"`
	...
}
```

https://github.com/edgexfoundry/device-sdk-go/blob/ba03d3229d39ccd3d083a66246eea2ffd8e01960/internal/provision/devices.go#L65
```
d := struct {
				DeviceList []dtos.Device `yaml:"deviceList"`
			}{}
```

Reply:
The original camera.toml.example has `DeviceList` pascal case, I only convert the file format to yaml here, no touch on word case inside the file. @weichou1229 are you expecting to change the `DeviceList` in camera.yaml.example to `deviceList`? @lenny-intel please advise

Reply:
@vli11 The full YAML content should be camelCase, not only `deviceList`.